### DISCUSSION TITLE
Lock Page option

### DISCUSSION THREAD

**Discussion by ithakaa** [docmost/docmost#399] [Ideas]
It would be nice to have a "lock page" option

**Comment by Philipinho** [comment]
What would locking a page mean to you, and who can lock and unlock a page?

There is an `is_locked` db column, but I have not gotten down to its technical details and implementation.

**Reply by kayvanaarssen** [reply]
That only users with permissions can unlock the page. If a page is locked it cannot be edited. Like if its a SOP / Process etc. It does not need to be edited (often) or even ever, like with versions etc.

**Comment by fuscodev** [comment]
I did it in PR #666

**Reply by adrian-dumitrescu-tfs** [reply]
Hmm does not seem to be in the current main...

**Reply by tajchert** [reply]
Any update? this feature is missing from current release.
